test/test.cc

```diff
@@ -2611,6 +2645,23 @@ TEST_F(ServerTest, MultipartFormData) {
   EXPECT_EQ(200, res->status);
 }
 
+TEST_F(ServerTest, MultipartFormDataMultiValue) {
```

Comment:
Could you rename `MultipartFormDataMultiValue` to `MultipartFormDataMultiFileValues`?